src/server/pfs/cmds/cmds.go

```diff
@@ -663,6 +671,40 @@ want to consider using commit IDs directly.
 			if err != nil {
 				return err
 			}
+
```

Comment:
Does this mean that tracing is only instrumented for the `put-file` call? Or can you use it with other calls?

Reply:
I see, that makes sense.